test/rust/basic/add_fail.rs

```diff
@@ -0,0 +1,9 @@
+#[macro_use]
+mod smack;
+use smack::*;
+// @expect error
```

Comment:
To make these consistent with C regressions, leave an empty line before and after `@expect` comment and other flags.